project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.5.2
+sbt.version=1.5.4
```

Comment:
Nit: would it make sense/be feasible to share this version b/w all 3 `build.properties` files somehow?

Reply:
Ofc symlink being the crudest way of doing that 😅

Reply:
Symlink won't work, because sbt copies project to temporary directory for testing

Reply:
On the other hand, I managed to merge to tests into one, using cross building